Modules/core/Model/CoreSpaceAccessOptions.php

```diff
@@ -28,18 +28,18 @@ public function __construct() {
     }
     
     public function getAll($id_space){
-        $sql = "SELECT * FROM core_space_access_options WHERE id_space=?";
+        $sql = "SELECT * FROM core_space_access_options WHERE id_space=? AND deleted=0";
         return $this->runRequest($sql, array($id_space))->fetchAll();
     }
     
-    public function set($id_space, $toolname, $module, $url){
+    public function set($id_space, $toolname, $module, $url, $deactivate=0) {
```

Comment:
why using delete param on set  and not calling delete (or a delete like based on toolname/module)

Reply:
ah yes, would be more coherent

Reply:
done